src/Traits/LogsActivity.php

```diff
@@ -310,6 +313,14 @@ function ($new, $old) {
                         return $new === $old ? 0 : 1;
                     }
 
+                    // Handels Date intervels comparsons since php cannot use spaceship
```

Comment:
Could you add 3 testcases for this new logic? The code was out of my head and in no way tested.

```
null => DateInterval
DateInterval#1 => DateInterval#2
DateInterval => null
```